frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -95,28 +95,51 @@ const getMetadata = (formData: DeployImageFormData) => {
 
 export const createDeployment = (
   formData: DeployImageFormData,
-  imageStreamName: string,
   dryRun: boolean,
 ): Promise<K8sResourceKind> => {
   const {
+    registry,
     project: { name: namespace },
     name,
-    isi: { ports },
+    isi: { image, ports },
     deployment: { env, replicas },
     labels: userLabels,
     limits: { cpu, memory },
+    imageStream: { image: imgName, namespace: imgNamespace, tag },
   } = formData;
 
+  const defaultAnnotations = {
+    ...annotations,
+    ...{
```

Comment:
There really isn't any value to spreading an object of static values. Just place them in one by one after the spread.

ie
```
{
  ...someOtherObject,
  keyA: valueA,
  keyInOtherObject: newValue,
  etc: etc,
}
```

Reply:
done